packages/components/result/src/result.vue

```diff
@@ -35,7 +35,7 @@ defineOptions({
 
 const props = defineProps(resultProps)
 
-const ns = useNamespace('result')
+const nsResult = useNamespace('result')
```

Comment:
There is no need to change the variable name.

Reply:
updated

Reply:
updated